gui-libs/libpanel/libpanel-1.0.1.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+VALA_MIN_API_VERSION="0.52"
+inherit gnome.org meson python-any-r1 vala virtualx
+
+DESCRIPTION="A dock/panel library for GTK 4"
+HOMEPAGE="https://gnome.pages.gitlab.gnome.org/libpanel/ https://gitlab.gnome.org/GNOME/libpanel"
+
+LICENSE="LGPL-2.1+"
+SLOT="1"
+IUSE="+introspection test +vala"
+REQUIRED_USE="vala? ( introspection )"
+
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
```

Comment:
This is a new package and I assume you have a amd64 computer. So drop all keywords apart from ~amd64